src/app/pages/product-item.page/product-item.page.component.html

```diff
@@ -0,0 +1 @@
+<section class="frame"><p>product-item.page works!</p></section>
```

Comment:
Is this correct?

Reply:
I would like to make this page in the next PR.